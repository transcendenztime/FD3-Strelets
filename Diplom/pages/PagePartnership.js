import React from 'react';

import { NavLink } from 'react-router-dom';


import '../styles/pages/Partnership.css';

class PagePartnership extends React.PureComponent {

  render() {

    return (
      <div  className = "partnership page">
        <h1 className="page-title">О компании:</h1>

        <div className = "breadcrumbs-container">
          <NavLink to="/" exact className="breadcrumbs">Главная </NavLink>
          <span> &rarr; </span>
          <NavLink to="/partnership" className="breadcrumbs">Сотрудничество</NavLink>
        </div>
        <hr />

        <section className="description">
          <blockquote>
            <div>Приглашаем Вас к взаимовыгодному сотрудничеству с нашей компанией.</div>
            <div>Мы рады предложить Вам широкий ассортимент товаров по самым низким ценам.</div>
            <div>На данный момент у нас более 5000 позиций.</div>
            <div>Мы обеспечим индивидуальный подход, гибкую ценовую политику, наилучшее качество продукции.</div>
            <div>По вопросам оптовых цен можно связаться по тел +375291643523.</div>
            <div>или оставив заявку на наш электронный адрес: sadmir@bk.ru</div>
            <div>Очень будем рады крепким и честным отношениям.</div>
            <div>С Уважением! Долина Растений</div>
          </blockquote>
        </section>
        
      </div>
    );
    
  }

}
    
export default PagePartnership;
    