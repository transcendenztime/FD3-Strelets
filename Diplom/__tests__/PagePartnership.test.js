import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

import PagePartnership from '../pages/PagePartnership';

test('внешний вид PagePartnership', () => {

    // создаём тестовую версию компонента
    const component = renderer.create(
      <Router>
          <PagePartnership />
      </Router>
    );
  
    // получаем снэпшот (HTML-снимок) компонента для сверки, что вёрстка не испортилась
    let componentTree=component.toJSON();
    expect(componentTree).toMatchSnapshot();
          
  });