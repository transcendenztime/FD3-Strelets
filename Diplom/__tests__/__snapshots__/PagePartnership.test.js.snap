// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`внешний вид PagePartnership 1`] = `
<div
  className="partnership page"
>
  <h1
    className="page-title"
  >
    О компании:
  </h1>
  <div
    className="breadcrumbs-container"
  >
    <a
      aria-current={null}
      className="breadcrumbs"
      href="/"
      onClick={[Function]}
      style={undefined}
    >
      Главная 
    </a>
    <span>
       → 
    </span>
    <a
      aria-current={null}
      className="breadcrumbs"
      href="/partnership"
      onClick={[Function]}
      style={undefined}
    >
      Сотрудничество
    </a>
  </div>
  <hr />
  <section
    className="description"
  >
    <blockquote>
      <div>
        Приглашаем Вас к взаимовыгодному сотрудничеству с нашей компанией.
      </div>
      <div>
        Мы рады предложить Вам широкий ассортимент товаров по самым низким ценам.
      </div>
      <div>
        На данный момент у нас более 5000 позиций.
      </div>
      <div>
        Мы обеспечим индивидуальный подход, гибкую ценовую политику, наилучшее качество продукции.
      </div>
      <div>
        По вопросам оптовых цен можно связаться по тел +375291643523.
      </div>
      <div>
        или оставив заявку на наш электронный адрес: sadmir@bk.ru
      </div>
      <div>
        Очень будем рады крепким и честным отношениям.
      </div>
      <div>
        С Уважением! Долина Растений
      </div>
    </blockquote>
  </section>
</div>
`;
